tests/utils.py

```diff
@@ -48,6 +48,12 @@ def patron_ref_builder(loan_pid):
     return {"ref": "{}".format(loan_pid)}
 
 
+# NOTE: Its on purpose `ref` and not `$ref` so it doesn't try to resolve
+def document_ref_builder(loan_pid):
+    """Return the $ref for item_pid."""
```

Comment:
`for the given loan_pid`

Can you please also fix it just above in `patron_ref_builder` and `item_ref_builder`? Or have a better comment?